dist/extension.html

```diff
@@ -5,24 +5,17 @@
   <meta charset="utf-8">
   <link rel="manifest" href="manifest.json">
 
-  <title></title>
+  <title>Calypso - Crypto rates</title>
 </head>
 
 <body id="cExtension">
   <div class="header">
-    <h1 class="header__title">Crypto-balance</h1>
+    <h1 class="header__title">Crypto rates</h1>
     <div id="app"></div>
   </div>
   <div id="rates" class="rates__container">
-    <!-- rates go in here... -->
   </div>
-  <!-- END Rates -->
-  <!-- <div class="footer">
-    <div id="refreshRates" class="refresh">
-      <a class="refresh__arrow" onclick="refreshRates(e);"></a>
-    </div>
-  </div> -->
 
-<script type="text/javascript" src="bundle.js?98dc4741224a63aa816a"></script></body>
+<script type="text/javascript" src="bundle.js?dd75db351dca3c17f4ed"></script></body>
```

Comment:
What's up with this query string? Why is it here?

Reply:
This string prevents bundle.js from caching. Isn't this a good thing?